src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1838,16 +1838,9 @@ Mavlink::task_main(int argc, char *argv[])
 
 	_interface_name = nullptr;
 
-#ifdef __PX4_NUTTX
-	/* the NuttX optarg handler does not
-	 * ignore argv[0] like the POSIX handler
-	 * does, nor does it deal with non-flag
-	 * verbs well. So we remove the application
-	 * name and the verb.
-	 */
+	// We don't care about the name and verb at this point.
```

Comment:
So we just happened to get away with this on posix because no one add expectations for argv entries at a hard coded index?

Reply:
Presumably.